che-plugins.yaml

```diff
@@ -172,3 +172,14 @@ plugins:
             path: /home/user/.local/share/buildkit
           - name: buildkit-lib
             path: /var/lib/buildkit
+  - repository:
+      url: https://github.com/qjebbs/vscode-plantuml
+      revision: v2.14.0
+    sidecar:
+      directory: plantuml
+      name: vscode-plantuml
+      memoryLimit: '512Mi'
+    aliases:
+      - qjebbs/vscode-plantuml
```

Comment:
We don't need an alias unless there are multiple versions of the same plugin, from the same repository. So this line is not needed.